content/en/blog/custom-metrics-autoscaling-new-deploy.md

```diff
@@ -0,0 +1,596 @@
++++
+title = "Autoscaling on Custom Metrics in New Deploy Functions"
```

Comment:
Let's change the title to "Autoscaling serverless functions with custom metrics"